docker/README.md

```diff
@@ -103,30 +114,86 @@ You may also want to modify the `-p 80:80` to utilize other ports instead. Refer
+$ docker run -i -t -d -p 80:80 --name mms -v /home/user/models:/models mms_image:latest
+# This runs the docker image in a detached mode
 ```
-Now that you have a prompt inside the container, the final step is to run the following command:
 
+To run the MMS with $HOSTNAME as its endpoint, or for `nginx's server_name` to be configured to `$HOSTNAME` run the following command.
 ```bash
-cd mms_docker_cpu && ./launch.sh
+$ echo $HOSTNAME
+user.ant.amazon.com
+
+$ docker run -i -t -d -p 80:80 --name mms -v /home/user/models:/models -e MMS_HOST=$HOSTNAME mms_image:latest
+```
+The above command lets you run inference with `$HOSTNAME` as 'server_name'.  
+   
+Verify that this image is running by running 
+```bash
+$ docker ps -a
+```
+You should see that the image is running.
+
+## Starting MMS in the docker instance
+To interact with the docker image named `mms` we run the following command
+```bash
+$ docker exec mms bash -c "mxnet-model-server.sh help"
+Usage:
+
+/mxnet_model_server/mxnet-model-server.sh [start | stop | restart | help] [--mms-config <MMS config file>]
+
+start        : Start a new instance of MxNet model server.
+stop         : Stop the current running instance of MxNet model server
+restart      : Restarts all the MMS worker instances.
+help         : Usage help for /mxnet_model_server/mxnet-model-server.sh
+--mms-config : Location pointing to the MxNet model server configuration file.
+To start the MxNet model server, run
+/mxnet_model_server/mxnet-model-server.sh start --mms-config <path-to-config-file\>
+
+To stop the running instnce of MxNet model server, run
+/mxnet_model_server/mxnet-model-server.sh stop
+
+To restart the running instance of MxNet model server, run
+/mxnet_model_server/mxnet-model-server.sh restart --mms-config <path-to-config-file\>
+```
+```bash
+# To start the MMS run the following
+$ docker exec mms bash -c "mxnet-model-server.sh start --mms-config /models/mms-app-cpu.conf"
+```
+
+Verify that the docker image is still running by running 
+```bash
+$ docker ps -a
 ```
 This will setup the MMS endpoint, gunicorn wsgi entry point, and nginx proxy_pass. 
-At this point you should see the typical MMS CLI output indicating that the server is running.
+At this point you should be able to run inference on `localhost` port `80`
 
 ### Running the MMS GPU Docker
 
 ```bash
-nvidia-docker run -it -p 80:80 mms_image_gpu:latest
+$ nvidia-docker run -i -t -d -p 80:80 --name mms -v /home/user/models/:/models mms_image_gpu:latest
 ```
 
-Now you should be inside the docker container at a bash prompt. The config file, `mms_app.conf` is located in the `mms_docker_gpu` folder. Run following command to launch the MMS service:
+To configure the nginx hostname to the $HOSTNAME, run the following command
+```bash
+$ echo $HOSTNAME
+user.ant.amazon.com
+```
+```bash
+$ nvidia-docker run -t -i -d -p 80:80 --name mms -v /home/user/models:/models -e MMS_HOST=$HOSTNAME mms_image_gpu:latest
```

Comment:
MMS_HOST: all our other env vars are MXNET_MODEL_SERVER...

Reply:
Changed this to MXNET_MODEL_SERVER_HOST